### ISSUE TITLE
linux64 bootloader requires glibc 2.14

### ISSUE THREAD

**Issue by ThomasWaldmann** [pyinstaller/pyinstaller#2160]
I tried to release borgbackup 1.1.0b1 (usual release process, building binaries on wheezy64, with recent pyinstaller from develop branch), but I got this:

```
(borg-env) vagrant@vagrant:/vagrant/borg$ ./borg.exe --version
../borg.exe: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by ../borg.exe)
```

```
(borg-env) vagrant@vagrant:/vagrant/borg$ objdump -T borg.exe|grep 2.14
0000000000000000      DF *UND*  0000000000000000  GLIBC_2.14  memcpy
```

Looks like memcpy in the 64bit binary requires a rather new glibc. Too new for debian wheezy64.

BUT: the same build process resulted in a 32bit binary on wheezy32. There, memcpy is from glibc 2.0.

Strange.

**Comment by ThomasWaldmann** [comment]
Guess that is a good reason to revert this one commit:

https://github.com/pyinstaller/pyinstaller/pull/2148/commits/1fab36c7108304db6c8fdae60b5143606a0534aa

So much for micro-optimizing stuff for no good reason.

**Comment by ThomasWaldmann** [comment]
Or, if the 64bit bootloader was built fundamentally differently than the 32bit one, build it in the same way.

**Comment by ThomasWaldmann** [comment]
https://www.win.tue.nl/~aeb/linux/misc/gcc-semibug.html < wtf

**Comment by ThomasWaldmann** [comment]
I tried with some small sample code to confirm: using memcpy creates the glibc 2.14 requirement, strcat is satisfied with glibc 2.2.5.

So, someone please revert the 3rd commit of that PR and recompile the stuff.

**Comment by ThomasWaldmann** [comment]
@htgoebel fixed by PR #2177. Also apply the other PR please, which fixes #2176.
